query-engine/connectors/sql-query-connector/src/filter_conversion.rs

```diff
@@ -563,6 +595,11 @@ fn default_scalar_filter(
             _ => comparable.not_in_selection(convert_values(fields, values)),
         },
         ScalarCondition::JsonCompare(_) => unreachable!(),
+        ScalarCondition::Search(value, _) => comparable.matches(match value {
+            PrismaValue::String(s) => s,
```

Comment:
You should probably use the try into transformers here to convert the value instead of matching.